sdk-extensions/autoconfigure/build.gradle.kts

```diff
@@ -52,7 +52,9 @@ testing {
         implementation(project(":exporters:logging-otlp"))
         implementation(project(":exporters:otlp:all"))
         implementation(project(":exporters:prometheus"))
-        implementation("io.prometheus:prometheus-metrics-exporter-httpserver")
+        implementation("io.prometheus:prometheus-metrics-exporter-httpserver") {
+          exclude(group = "io.prometheus", module = "prometheus-metrics-exposition-formats")
```

Comment:
What is the goal of this exclusion?

Reply:
it aligns the classpath with what is used for compile - see https://github.com/open-telemetry/opentelemetry-java/blob/1efe21585ebfd681154b2a6ff51538373698ec1e/exporters/prometheus/build.gradle.kts#L15